testapp/build.gradle

```diff
@@ -53,7 +53,8 @@ android {
             subscriptionKey         : property("HOST_APP_SUBSCRIPTION_KEY") ?: "test-subs-key",
             adMobAppId              : property("ADMOB_APP_ID") ?: "",
             appName                 : defaultAppName + " DEBUG",
-            ratEndpoint             : property("ANALYTICS_ENDPOINT") ?: "https://www.example.com/"
+            ratEndpoint             : property("ANALYTICS_ENDPOINT") ?: "https://www.example.com/",
+            sslPinningPublicKey     : property("CERTIFICATE_PUBLIC_KEY") ?: ""
```

Comment:
The public key should be different for Staging and Prod. So you need to setup two variables for this: 
`STG_CERTIFICATE_PUBLIC_KEY` and `PROD_CERTIFICATE_PUBLIC_KEY`